mne_qt_browser/_pg_figure.py

```diff
@@ -1809,19 +1815,22 @@ def __init__(self, main, title="Settings", **kwargs):
         ch_scaling_layout = QFormLayout()
         self.ch_scaling_spinboxes = {}
 
-        # Get all unique channel types
+        # Get all unique channel types and allow scaling
         ordered_types = self.mne.ch_types[self.mne.ch_order]
         unique_type_idxs = np.unique(ordered_types, return_index=True)[1]
         ch_types_ordered = [ordered_types[idx] for idx in sorted(unique_type_idxs)]
         for ch in ch_types_ordered:
             if ch in self.mne.unit_scalings.keys():
                 ch_spinbox = QDoubleSpinBox()
+                ch_spinbox.setMinimumWidth(100)
                 ch_spinbox.setRange(-float("inf"), float("inf"))
```

Comment:
I don't think you should be able to set negative values. Zero is probably bad, too, but this is at least a step in the right direction:
```suggestion
                ch_spinbox.setRange(0, float("inf"))
```